src/vm/assembly.cpp

```diff
@@ -1219,7 +1219,9 @@ Module * Assembly::FindModuleByTypeRef(
             if (pModule->HasBindableIdentity(tkType))
 #endif// FEATURE_COMINTEROP
             {
+#ifndef CROSSGEN_COMPILE
```

Comment:
Can we come here at runtime as well if somebody tries to run assembly with WinRT reference on Unix?

Reply:
Yeah ok I'll just remove it altogether. I figured there was value in keeping this assert around but at runtime we'll get a bind failure and that's better behavior.